File: Java/Scripts/ArraySummation.java
import java.util.Scanner;

class Solver{
    int minNumber, maxNumber, numSum, numLen, Arr[];
    Solver(int []Arr){
        this.Arr = Arr;
        if (this.Arr[0] > this.Arr[1]){
            this.minNumber = this.Arr[1];
            this.maxNumber = this.Arr[0];
        }else {
            this.minNumber = this.Arr[0];
            this.maxNumber = this.Arr[1];
        }
        this.numLen = (this.maxNumber - this.minNumber)+1;
    }
    int getSum(){
        int maxPlusMin = this.maxNumber + this.minNumber;
        int sum = (this.numLen*maxPlusMin)/2;
        return sum;
    }
}


public class ArraySummation{
    public static void main(String[] args) {
        int []Arr = new int[2];
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the elements:");
        for (int i=0; i < 2; i++) {
            Arr[i] = sc.nextInt();
        }
        long startTime = System.nanoTime();
        Solver solver = new Solver(Arr);
        int sum = solver.getSum();
        long endTime = System.nanoTime();
        System.out.println("The sum is: " + sum);
        System.out.println("Time taken: " + (endTime - startTime)+" nanoSeconds.");
        sc.close();
    }
}

File: Python3/Scripts/ArraySummation.py
from time import time

class Solver:
    def __init__(self, Arr:list) -> None:
        self.Arr:list = Arr
        if (self.Arr[0] > self.Arr[1]):
            self.minNumber:int = self.Arr[1]
            self.maxNumber:int = self.Arr[0]
        else: 
            self.minNumber:int = self.Arr[0]
            self.maxNumber:int = self.Arr[1]
        self.numLen:int = (self.maxNumber - self.minNumber)+1
    def getSum(self) -> int:
        self.maxPlusMin:int = (self.maxNumber+self.minNumber)
        add:int = int((self.maxPlusMin*self.numLen)/2)
        return add

def main() -> None:
    Arr:list = list()
    print ("Enter the elements Values: ")
    for _ in range(0,2): Arr.append(int(input()))
    startTime:float = time()
    solver:Solver = Solver(Arr)
    sum:int = solver.getSum()
    endTime:float = time()
    print ("the Sum is: {}".format(sum))
    print ("time taken: {} nanoSeconds".format((endTime-startTime)*1000000000))

if __name__ == "__main__":
    main()